rust/datafusion/src/execution/context.rs

```diff
@@ -326,7 +326,9 @@ impl ExecutionContext {
                 match expr {
                     &Expr::Literal(ref scalar_value) => {
                         let limit: usize = match scalar_value {
-                            ScalarValue::Int8(limit) if *limit >= 0 => Ok(*limit as usize),
+                            ScalarValue::Int8(limit) if *limit >= 0 => {
```

Comment:
This is the cause of the failure, might be because I ran `fmt` before rebasing. Oddly I didn't see it in my commited files at the time